ui/libs/types/src/lib/operators.ts

```diff
@@ -0,0 +1,12 @@
+import {map, Observable} from "rxjs";
+
+/**
+ * Apply map to observable and map on array using provided function
+ *
+ * @param _func
+ */
+export function mapArray<T, V>(_func: (element: T) => V) {
+  return function(source: Observable<T[]>): Observable<V[]> {
+    return source.pipe(map(array => array.map(_func)));
+  }
+}
```

Comment:
Are you sure the operator is simplifying anything? 

`map(array => array.map(_func))` is replaced with `mapArray(_func)`

I believe the added value is too small to use and remember about such a tool in future development.